datacite/version.py

```diff
@@ -18,4 +18,4 @@
 # Do not change the format of this next line. Doing so risks breaking
 # setup.py and docs/conf.py
 
-__version__ = "0.1.0.dev20140223"
+__version__ = "0.1.1.dev20150225"
```

Comment:
It's not possible to post tag it? I can take it out, just thought it's easier to have it as one logical part.